src/components/Headline/docs/Headline.mdx

```diff
@@ -54,4 +64,5 @@ provide an accessible experience for screen reader users.
 
 <Playground>
```

Comment:
While we're at it, can we change the headline for this section to `## Examples` instead of playground, like with our other documentation pages?